pkg/daemon/daemon.go

```diff
@@ -720,15 +747,57 @@ func (dn *Daemon) syncNode(key string) error {
 	}
 
 	if ufc != nil {
+
+		err = upgrademonitor.GenerateAndApplyMachineConfigNodes(
+			&upgrademonitor.Condition{State: mcfgalphav1.MachineConfigNodeUpdated, Reason: string(mcfgalphav1.MachineConfigNodeUpdated), Message: fmt.Sprintf("Node %s needs an update", dn.node.GetName())},
+			nil,
+			metav1.ConditionFalse,
+			metav1.ConditionFalse,
+			dn.node,
+			dn.mcfgClient,
+			dn.featureGatesAccessor,
+		)
+		if err != nil {
+			klog.Errorf("Error making MCN: %w", err)
+		}
+		/*
+			annos := map[string]string{
```

Comment:
Is this intentional or need cleaning up?

Reply:
need to do some cleaning, thanks for catching this one